phpBB/docs/events.md

```diff
@@ -281,29 +281,29 @@ acp_overall_header_stylesheets_after
 * Purpose: Add assets after stylesheets within the `<head>` tags in the ACP.
 Note that INCLUDECSS will not work with this event.
 
-acp_permission_forum_copy_src_forum_append
+acp_permission_forum_copy_dest_forum_append
 ===
 * Location: adm/style/permission_forum_copy.html
 * Since: 3.1.7-RC1
-* Purpose: Add content after the sourse forum select form
+* Purpose: Add content after the destiny forum select form
```

Comment:
destination

Reply:
Heh, yeah. Not sure how that happened :)